### PR TITLE
Add :skip_audience option

### PR DESCRIPTION
**Pull request by swordfish444**
# Summary

Adds an option to the SAML Response class, to skip the assertions during `#validate_audience`. 

## Why this matters?

Some Service providers want to relax validation, and have this control via configuration. This was first surfaced in #515 and @pitbulk requested opening a PR to introduce it. Here's the PR =)

### CODE REVIEW

**Comment by swordfish444** [comment]
@pitbulk Can I get your review/approval on this please?

File: lib/onelogin/ruby-saml/response.rb
Comment (pitbulk) [inline lib/onelogin/ruby-saml/response.rb:607]: can you do it before anything else?

**Comment by pitbulk** [comment]
Can you update the [README](https://github.com/onelogin/ruby-saml#the-initialization-phase) (Like 294) to mention this skip option?

File: lib/onelogin/ruby-saml/response.rb
Comment (swordfish444) [reply lib/onelogin/ruby-saml/response.rb:607]: @pitbulk Sure, I was following the format done in `#validate_destination`, otherwise I would have defaulted to it being first check.

File: README.md
Comment (swordfish444) [inline README.md:295]: @pitbulk readme has already been updated. See here ☝️

File: lib/onelogin/ruby-saml/response.rb
Comment (swordfish444) [reply lib/onelogin/ruby-saml/response.rb:606]: @pitbulk done 👍

**Comment by swordfish444** [comment]
@pitbulk ready for re-review. Not sure if you're setup to monitor requests for re-review, but I triggered that as well. APologies for multiple notifications in the event you are!

If there are no further questions/feedback, can we get this approved/merged and put an ETA on when the next release can be pushed out?

**Review by pitbulk — APPROVED** [review]

**Comment by pitbulk** [comment]
I'm releasing java-saml toolkit and ruby-saml will be the next, I hope to release it in 1-2 weeks

**Pull request merged**
